tests/acceptance/query-error-test.js

```diff
@@ -15,7 +15,8 @@ moduleForAcceptance('Acceptance | query error', {
   suppressLogging: true,
 
   beforeEach() {
-    server = mockAPI(RESULTS.MULTIPLE, COLUMNS.BASIC);
+    server = mockAPI(COLUMNS.BASIC);
```

Comment:
If we are injecting the mock into here, can we do that for the pretender API as well, so we don't have to import it as well? It also looks consistent.

Reply:
Done.